package/yast2-firstboot.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Fri Mar 13 13:15:39 UTC 2020 - Josef Reidinger <jreidinger@suse.com>
+
+- remember correctly keyboard if setting in keyboard language
```

Comment:
np: would maybe reword it a little bit but it is ok as it is

- remember the keyboard selection in the keyboard language dialog when the language is selected after the keyboard`